modules/fontrendering/include/modules/fontrendering/textrenderer.h

```diff
@@ -269,7 +241,11 @@ class IVW_MODULE_FONTRENDERING_API TextRenderer {
      */
     int getBaseLineDescender() const;
 
+    void configure(const FontSettings &settings);
+
 protected:
+    static std::shared_ptr<Shader> getShader();
```

Comment:
Why its not related

Reply:
Saw that the same logic was called getDefaultShader in another part of the PR. But that might have been because one could override the shader, which is not possible here